WordPress/Classes/ViewRelated/System/ForcePopoverPresentationPopoverControllerDelegate.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+/// This delegate forces popover presentation even on iPhone / in compact size classes
+class ForcePopoverPresentationPopoverControllerDelegate: NSObject, UIPopoverPresentationControllerDelegate {
```

Comment:
This class name is a bit intense 😄, just spitballing here, but what about `ForcedPopoverPresentation` since it kind of acts as more of a helper than an object. 🤔